modules/andes-core/broker/src/main/java/org/wso2/andes/server/AMQBrokerManagerMBean.java

```diff
@@ -68,6 +67,9 @@
 
     private final VirtualHostImpl.VirtualHostMBean _virtualHostMBean;
 
+    private static AndesContextStore contextStore;
+    public static boolean exclusiveConsumer;
```

Comment:
What is the purpose of having this public variable ? . By any chance can we make it a local variable ?